packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -593,8 +592,7 @@ class EditorFormComponent extends Component<EditorFormComponentProps,EditorFormC
   renderVersionSelect = () => {
     const { classes, document, currentUser } = this.props
 
-    if (!userHasCkEditor(currentUser)) return null
-
+    if (!currentUser?.isAdmin) return null
```

Comment:
It looks like we'd been giving version select to everyone who had enabled ckEditorBeta before, and now we are removing that? 

I think we should probably just give everyone version select, although not necessarily in this particular deploy

Reply:
We definitely shouldn't give anyone the current version select component. It's horribly broken, sometimes deletes your content, and doesn't even work reliably for admins. We should at some point actually push a functioning version-select component, but that's a big push that will probably take multiple days.

Reply:
Let me know when this merge actually happens. I need to do stuff for Tagging Dashboard switchover.